recipes/libusb/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "libusb" %}
+{% set version = "1.0.22" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 'https://github.com/libusb/libusb/releases/download/v{{ version}}/{{ name }}-{{ version }}.tar.bz2'
+  sha256: 75aeb9d59a4fdb800d329a545c2e6799f732362193b465ea198f2aa275518157
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libusb') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('libusb') }}  # [linux]
```

Think this should be...

```suggestion
    - {{ cdt('libudev-devel') }}  # [linux]
```

---

Hahaha I'm so sorry, it's 40 degrees here and it seems my brain's been melting. Will fix thus.

---

No worries. 🙂

---

Fixed in 2ade825